JWBestPracticeApps/FeedTableViewController/FeedTableViewController/Feed/FeedViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  FeedViewModel.swift
+//  FeedTableViewControllerV4
+//
+//  Created by Amitai Blickstein on 5/2/22.
+//
+
+import Foundation
+import JWPlayerKit
+
+/// All hard-coded and dynamic values required by the table view.
+/// Since there is only one d-source for a table view, this is a singleton.
+final class FeedViewModel {
+    // Singleton pattern
+    static let shared = FeedViewModel()
+    private init() {}
+    
+    // Dynamic values
+    private var items = [JWPlayerItem]()
```

Comment:
"Dynamic Values" does not say what this property is. The fact that it is a 'var' and not a 'let' makes it dynamic. Perhaps a better description is "The player items in the feed."

Reply:
Erred on the side of (somewhat verbose) clarity.